### PR TITLE
Adding the common requirements to the tomtom image

### PR DESCRIPTION
**Pull request by aerostitch**
This way we can remove the need to add the `setup_common_requirements.sh` and save time not reinstalling things every time we build.

@pgrandin question that might seem stupid but why do we recompile the libs from scratch instead of using the ones in the repo for tomtom? (like libxml2 or zlib for example)

### CODE REVIEW

**Review by jkoan — APPROVED** [review]
Content approved, but waiting for the question to be answered before merge ;)

**Comment by pgrandin** [comment]
> why do we recompile the libs from scratch instead of using the ones in the repo for tomtom? (like libxml2 or zlib for example)

I think that it was related to the need of one specific version, IIRC.

**Pull request merged**
